client/src/components/profileInfoEdit/ProfileInfoEdit.js

```diff
@@ -20,6 +20,8 @@ import { updateTeam } from '../../actions/team';
 const ProfileInfoEdit = () => {
   const dispatch = useDispatch();
 
+  const uploadedImage = React.useRef(profilePic); // To be referred to server default profile image
```

Comment:
Although the backend is not implemented, would this variable be retrieved from the redux state cause the team image is one of the properties in the teamInfo object?

Reply:
+1, we should implement the corresponding Redux functionality in this PR.

Reply:
> > > > Nice quick PR! Great work!
> > > > Is the default picture empty only for me?
> > > > ![Screen Shot 2021-09-06 at 8 54 29 pm](https://user-images.githubusercontent.com/63642262/132207930-961cb8eb-0602-45d0-a92e-927e8d1c62bc.png)
> > > 
> > > 
> > > Yeah it's not fetching anything at the moment. So we'll probably have to set a default pic in the backend logic.
> > 
> > 
> > A follow-up question, as the default pic is not working and it will be fetched from the backend, can the `import profilePic from '../../images/profilepic.jpg';` on line14 be moved and just set null on line 23? `const uploadedImage = useRef(null);`
> 
> I feel like the default pic shouldn't come from backend, we can just use the pic we currently have `../../images/profilepic.jpg;`. This way we avoid having to make an extra get request to backend to get the default pic.

Sure that's alright then, I just added a simple if statement to check if image is undefined from redux state.

Reply:
Empty profilePic redux state:
![image](https://user-images.githubusercontent.com/53892953/132434035-16e5bcee-f1a4-4d1f-9b94-b2655b927205.png)

Preview of image upload:
![image](https://user-images.githubusercontent.com/53892953/132434110-96e31ba2-5bb3-437e-aac5-77f2b374c86f.png)